### ISSUE TITLE
x509: failed to load system roots and no roots provided

### ISSUE THREAD

**Issue by rthewhite** [hypriot/rpi-traefik#20]
Hi,
I'm trying to run this container on my ARM based Kubernetes cluster and would like to use the Lets Encrypt functionality from Traefik but I'm running into the issue that all calls from within the container to https urls are failing:

```
x509: failed to load system roots and no roots provided
```

This happens for both the version check when the container starts:

```
"Error checking new version: Get https://update.traefik.io/repos/containous/traefik/releases: x509: failed to load system roots and no roots provided"  
```

As well as for the the calls to Lets Encrypt to generate the ACME account etc. 

Probably we need to install the ca-certificates in the alpine base image to resolve this?

**Comment by firecyberice** [comment]
Fixed with #22

**Comment by rthewhite** [comment]
Thanks 👍
